targets/TARGET_NORDIC/TARGET_NRF5x/us_ticker.c

```diff
@@ -110,7 +110,9 @@ void us_ticker_set_interrupt(timestamp_t timestamp)
 {
     core_util_critical_section_enter();
 
-    nrf_timer_cc_write(NRF_TIMER1, NRF_TIMER_CC_CHANNEL0, timestamp & 0xFFFF);
+    const uint32_t counter_mask = ((1 << US_TICKER_COUNTER_BITS) - 1);
```

Comment:
Will this bit-shift always work? I was expecting this:

`((1ULL << US_TICKER_COUNTER_BITS) - 1)`

Reply:
Fixed as suggested.